integration/eth2network/pos_eth2_network_test.go

```diff
@@ -34,6 +36,12 @@ func TestEnsureBinariesAreAvail(t *testing.T) {
 }
 
 func TestStartPosEth2Network(t *testing.T) {
+	testlog.Setup(&testlog.Cfg{
+		LogDir:      "../.build/noderunner/",
+		TestType:    "TestStartPosEth2Network",
+		TestSubtype: "test",
+		LogLevel:    log.LvlInfo,
```

Comment:
Is this intentional?

Reply:
yes. It was necessary for a previous change